lib/app/constants/region_data.dart

```diff
@@ -281,3 +281,382 @@ List<Map<String, dynamic>> cityRegionMap = [
     ]
   },
 ];
+// TODO: cityRegionMap 추후 삭제 예정
+
+enum CityEnum {
+  seoul('서울'),
+  incheon('인천'),
+  busan('부산'),
+  daejeon('대전'),
+  daegu('대구'),
+  gwangju('광주'),
+  ulsan('울산'),
+  jeju("제주"),
+  sejong('세종'),
+  gangwon('강원도'),
+  gyeonggi('경기도'),
+  gyeongsangnam('경상남도'),
+  gyeongsangbuk('경상북도'),
+  chungcheongnam('충청남도'),
+  chungcheongbuk('충청북도'),
+  jeollanam('전라남도'),
+  jeollabuk('전라북도');
+
+  final String label;
+  const CityEnum(this.label);
+
+  static CityEnum? fromServerData(String? value) {
+    if (value == null) return null;
+    try {
+      final camelCase = value.toLowerCase();
+      return CityEnum.values.firstWhere(
+        (city) => city.name == camelCase,
+      );
+    } catch (e) {
+      return null;
+    }
```

Comment:
```suggestion
    final lowerCaseValue = value.toLowerCase();
    return CityEnum.values.firstWhereOrNull(
      (city) => city.name == lowerCaseValue,
    );
```
1. camelCase는 이름이 맞지 않는 것 같아요~
2. 모든 발생할 수 있는 exception을 잡기보다 없을 경우 예외처리가 명시적인 method를 사용하면 좋을 것 같습니다. (다른 부분들도 전체적으로 검토 부탁드려요~)
3. server에서 받아오는 데이터일까요? + server에서 한글로 데이터를 내려주는건가요?

Reply:
1. 넵 수정하겠습니다!
2. 네~
3. 서버에서 받아오는 데이터입니다. 데이터값은 노션 > ETC. > 데이터 페이지에 명시되어있습니다
서버에서는 enum 값으로 내려줍니다!